### PR TITLE
Improve installation documentation

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -314,43 +324,50 @@ Ensure that you have at least 30GB of free space.
...
 ```
 
 _Windows 10:_ no action needed, should work out of the box.
 
-In Android Studio open the project in `android/` directory.
+### Create a device
 
-Setup a virtual device to use [emulator](https://developer.android.com/studio/run/emulator) ("Tools > Device Manager") or [use a hardware device for debugging](https://developer.android.com/studio/run/device). If using an emulator, make sure to choose a system image with API Level 31 and ABI _x86_64_.
+Setup a virtual device to use [emulator](https://developer.android.com/studio/run/emulator) ("Tools > Device Manager") or [use a hardware device for debugging](https://developer.android.com/studio/run/device).
+If using an emulator, make sure to choose a system image with API Level 31 and ABI _x86_64_.
```
Comment (biodranik) [inline docs/INSTALL.md:367]: ...The same api level for the SDK you installed?

Use ABI x86_64 for Intel-based processors and arm64-v8a for ARM-based processors (e.g. M1/M2 Macs)

File: docs/INSTALL.md [hunk 2]
```diff
@@ -314,43 +324,50 @@ Ensure that you have at least 30GB of free space.
...
 
-Android Studio has issues in parsing C++ part of the project - one of these applications should be used: [Qt Creator](https://www.qt.io/product/development-tools), [Xcode](https://developer.apple.com/xcode/), [CLion](https://www.jetbrains.com/clion/). For Xcode it is required to run `cmake . -g Xcode` to generate project files, while CLion and QT Creator can import CMakeLists.txt.
+Android Studio has issues in parsing C++ part of the project - one of these applications should be used: [Qt Creator](https://www.qt.io/product/development-tools), [Xcode](https://developer.apple.com/xcode/), [CLion](https://www.jetbrains.com/clion/).
```
Comment (biodranik) [inline docs/INSTALL.md:369]:
```suggestion
Android Studio has issues in parsing the C++ part of the project, please let us know if you know how to resolve it. As a workaround, for working C++ suggestions, you may use:
- [Qt Creator](https://www.qt.io/product/development-tools)
- [Xcode](https://developer.apple.com/xcode/)
- [CLion](https://www.jetbrains.com/clion/)
```

**Review by biodranik — CHANGES_REQUESTED** [review]
Thanks! A few comments.

File: docs/INSTALL.md [hunk 3]
```diff
@@ -89,6 +89,14 @@ Install Cmake (**3.22.1** minimum), Boost, Qt 6 and other dependencies.
...
 
+_Ubuntu 20.04:_
+
+Because Ubuntu 20.04 only offers Qt 5, you need to add a PPA to get Qt 6.2:
+
+```bash
+sudo add-apt-repository -y ppa:savoury1/qt-6-2
+```
+
 _Ubuntu 20.04, 22.04:_
```
[the remark this answers is not in the document]
Comment (simonschaufi) [reply docs/INSTALL.md:100]: It's correct, it works with the PPA together

File: docs/INSTALL.md [hunk 4]
```diff
@@ -314,43 +324,50 @@ Ensure that you have at least 30GB of free space.
...
 
 _macOS:_
+
 ```bash
 ./tools/android/set_up_android.py --sdk $HOME/Library/Android/Sdk
 ```
 
 _Windows 10:_ no action needed, should work out of the box.
 
-In Android Studio open the project in `android/` directory.
```
[the remark this answers is not in the document]
Comment (simonschaufi) [reply docs/INSTALL.md:349]: I moved it to another place that you so the steps in the right order.

File: docs/INSTALL.md (hunk 1 above)
Comment (simonschaufi) [reply docs/INSTALL.md:367]: I did exactly what it told me to do. I will adjust it for M processors

File: docs/INSTALL.md [hunk 5]
```diff
@@ -89,6 +89,14 @@ Install Cmake (**3.22.1** minimum), Boost, Qt 6 and other dependencies.
 
 Installing *ccache* can speed up active development.
 
+_Ubuntu 20.04:_
+
+Because Ubuntu 20.04 only offers Qt 5, you need to add a PPA to get Qt 6.2:
```
[the remark this answers is not in the document]
Comment (simonschaufi) [reply docs/INSTALL.md:94]: Well, this is what the PPA says itself: https://launchpad.net/~savoury1/+archive/ubuntu/qt-6-2/+index?batch=75&memo=150&start=150: "Qt 6.2.x - backports (Xenial & newer)". What do you suggest?

[the remark this answers is not in the document]
Comment (biodranik) [reply docs/INSTALL.md:94]: I suggest to mention that any version of Qt6 should work.

[the remark this answers is not in the document]
Comment (simonschaufi) [reply docs/INSTALL.md]: Done.

**Review by biodranik — APPROVED** [review]
Thank you!

**Pull request merged**